components/accelerator/nvidia/nccl/dmesg_matcher.go

```diff
@@ -22,23 +24,24 @@ func HasNCCLSegfaultInLibnccl(line string) bool {
 	return false
 }
 
-func Match(line string) (name string, message string) {
+func Match(line string) (eventName string, regex string, message string) {
```

Comment:
let's not return the regex unless it's necessary.

Reply:
@eahydra Done, PTAL